parsers/json_with_nested_key=value.conf

```diff
@@ -0,0 +1,28 @@
+//{"time-generated":"2024-03-12T12:16:33.959116+00:00", "event-id":"ABC123", "hostname":"test01", "fromhost":"192.168.1.1", "priority-text":"test.info", "severity":5, "program-name":null, "source":"foo bar", "message":" ABC01 03\/12\/2024 08:16:33 AM LogName=Microsoft-Windows-WinRM\/Operational EventCode=123 EventType=1 ComputerName=ABC01.domain.com User=NOT_TRANSLATED SourceName=Microsoft-Windows-Windows Remote Management Type=Information RecordNumber=12345 TaskCategory=Response handling OpCode=Stop Message=WSMan operation Enumeration completed successfully"}
+
+{
+  patterns: {
+    valPattern: ".+?(?=\\s\\w+=\\w+)"
```

Comment:
Positive Lookaheads are not supported AFAIK. 

I think what is making this work is the .+?

Which i think is pretty close to .*